pico/api/src/test/java/io/helidon/pico/api/PriorityAndServiceTypeComparatorTest.java

```diff
@@ -14,17 +14,17 @@
  * limitations under the License.
  */
 
-package io.helidon.pico;
+package io.helidon.pico.api;
 
 import java.util.ArrayList;
 import java.util.Comparator;
 import java.util.List;
 
 import io.helidon.common.Weighted;
 import io.helidon.common.Weights;
-import io.helidon.pico.testsubjects.PicoServices1Provider;
-import io.helidon.pico.testsubjects.PicoServices2Provider;
-import io.helidon.pico.testsubjects.PicoServices3Provider;
+import io.helidon.pico.api.testsubjects.PicoServices1Provider;
```

Comment:
Please fix here (and in any other class that has the same problem)

Reply:
this is expected as the test packages changed as well.